ftl/cache_cloudbuild.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - name: gcr.io/gcp-runtimes/${LANGUAGE}_cache_image:latest
+    args: [
+      '--language', '${LANGUAGE}',
+      '--packages',
+      '${PACKAGES}',
```

Comment:
Could you leave a comment about this format? Is it space separated, comma separated, etc.